src/client/components/forms/profile.js

```diff
@@ -123,6 +123,10 @@ class ProfileForm extends React.Component {
 			initialBirthDate = `${year}-${month}-${day}`;
 		}
 
+		function birthdayValidation(value) {
```

Comment:
I'd suggest `isValidUserBirthday` or the like. That makes it clear that it returns a bool and exactly what its purpose is.